app/editor/src/features/content/form/interfaces/IContentForm.ts

```diff
@@ -20,7 +22,9 @@ export interface IContentForm {
   status: ContentStatusName;
   contentType: ContentTypeName;
   ownerId: number | '';
+  owner: IUserModel | undefined;
```

Comment:
Rewrite to stay consistent with convention used everywhere else `owner?: IUserModel` and `product?: IProductModel`

Reply:
Updated, thanks